ros2msg/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2msg</name>
+  <version>0.0.0</version>
+  <description>
+    The msg command for ROS 2 command line tools.
+  </description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <license>Apache License 2.0</license>
+
+  <depend>ros2cli</depend>
+
+  <exec_depend>ament_index_python</exec_depend>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
```

Comment:
I don't think we ever installed this plugin because not available from debian packages (only via pip like `import-order`). Can't find any trace of it in the ci scripts either... Is there a PR I missed that was adding it ?